test/fixtures.ts

```diff
@@ -239,7 +240,7 @@ async function vestingRewardsFixtureFunction(this: Mocha.Context) {
     this.epochSize
   );
 
-  await stakerValidatorSet.stake({ value: this.minStake });
+  await stakerValidatorSet.stake({ value: hre.ethers.utils.parseEther("1000") });
```

Comment:
Why is this needed?

Reply:
Well, it is actually not needed as we discussed to just change the penalty and reporter reward in the Inspector contract.